libs/command/workspace_client.go

```diff
@@ -0,0 +1,22 @@
+package command
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go"
+)
+
+func SetWorkspaceClient(ctx context.Context, w *databricks.WorkspaceClient) context.Context {
+	if v := ctx.Value(workspaceClientKey); v != nil {
```

Comment:
I think it might okay to do so, for example, overriding with a different workspace client or smth like this, any thoughts?

Reply:
I like having this restriction because ideally every command should only configure a working workspace client once. This keep reasoning about auth simple and would force the upstream code to be designed better.

We can ofcourse remove it if we ever have a usercase where we need to override with a different workspace.